src/java/org/apache/cassandra/streaming/messages/IncomingStreamMessage.java

```diff
@@ -44,18 +43,19 @@ public IncomingStreamMessage deserialize(DataInputPlus input, int version) throw
             ColumnFamilyStore cfs = ColumnFamilyStore.getIfExists(header.tableId);
             if (cfs == null)
                 throw new StreamReceiveException(session, "CF " + header.tableId + " was dropped during streaming");
-
             try
             {
                 IncomingStream incomingData = cfs.getStreamManager().prepareIncomingStream(session, header);
                 incomingData.read(input, version);
 
                 return new IncomingStreamMessage(incomingData, header);
             }
-            catch (Throwable t)
+            catch (Exception | Error e)
```

Comment:
Not using `Throwable` because you want to exclude non `java.lang` subclasses?

Reply:
no real reason, I know some java style guides ask not to do Throwable and that Exception | Error does the same... I should revert